stumpy/stumped.py

```diff
@@ -248,13 +248,20 @@ def stumped(dask_client, T_A, m, T_B=None, ignore_trivial=True, normalize=True,
                 T_B_subseq_isconstant_future,
                 diags_futures[i],
                 ignore_trivial,
+                1,
```

Comment:
Are you planning to add `k` as an input param later?

Reply:
Should I take care of `stumped` and `scrump` right now? Or should I wait to handle the `stump` and its non-normalized version first? and then (after adding unit test ?) take care of those two functions by replacing `1` with `k`?

Reply:
I think we should handle `stump` first. Then we handle `stumped` and `scrumped`. Finally, we do the non-normalized versions. I'll leave it up to you if you'd like to do these all in this PR or in three separate PRs

Reply:
I think one PR is better so that I can better keep track of things all in one PR.